pkg/networkmanager/interfaces.js

```diff
@@ -2421,7 +2425,7 @@ PageNetworkInterface.prototype = {
             $('#network-interface-mac').text(mac);
         }
 
-        this.device_onoff.onoff("disabled", !iface);
+        this.device_onoff.onoff("disabled", !!(!iface || (dev && dev.State == 20)));
```

Comment:
Comment: Disable the On/Off button for interfaces that we don't know about at all, and for devices that NM declares to be unavailable.  Neither can be activated.